build-images/Earthfile

```diff
@@ -0,0 +1,5 @@
+VERSION 0.8
+
+build:
+  FROM DOCKERFILE --target build .
```

Comment:
I still think we'll avoid a lot of practical headache publishing this image

Reply:
I intend to publish (in fact have already published the wasi-sdk part). But right now the only issue should be cold start?